qml/Sim.qml

```diff
@@ -116,9 +114,8 @@ Item {
                     onClicked: {
 
                        var phoneNumbers = PhoneNumber.PhoneUtils.matchInText(newNumber.text, getCountryCode())
-                       if (phoneNumbers.length===0) {
+                       if (phoneNumbers.length===0 || phoneNumbers[0]!=="+") {
```

Comment:
here phoneNumbers[0]!=="+" will return always true because phoneNumbers[0] is the full number, not the first character of the number

Reply:
can you remove that check as i found better solution with already existing components